tests/orquestra/integrations/cirq/simulator/qsimsimulator_test.py

```diff
@@ -93,6 +93,19 @@ def test_get_wavefunction(self):
             wavefunction.amplitudes[7], (1 / np.sqrt(2) + 0j), atol=10e-15
         )
 
+    def test_normalization(self):
+        # Given
+        simulator1 = QSimSimulator()
```

Comment:
If you're testing a particular param it's good to be explicit and not rely on defaults, cause defaults might change in future:
```suggestion
        simulator1 = QSimSimulator(normalize_wavefunction=False)
```